### DISCUSSION TITLE
How to disable all or filtered writeable parameters?

### DISCUSSION THREAD

**Discussion by michaelschwarz** [wills106/homeassistant-solax-modbus#470] [Q&A] [answered]
Hi,

I didn't find a way on how to disable all/filtered writeable parameters. Background is that I am at least at the very beginning don't want to change something unintentionally, happenes quite easily using the mobile view of Home Assistant.

My inverter is a QCells HYB-G3 with PocketWifi and TCP direct.

Thanks!
Michael

**Comment by wills106** [comment, marked as answer]
There isn't a way to disable them all in one go.

You will need to hide / manually disable any Button, Number & Select entities.

Edit: Or you could have entities you can change on a separate dashboard page?

**Comment by michaelschwarz** [comment]
Yes, that works for me, however many integrations offer some global settings to have entities not being created (i.e., in HomeKit you can export just lights). Given the fact that writing system relevant settings is dangerous, it would be good to have these even not created.

Maybe it is just a nice to have option, but hiding them one-by-one is working for me.
